scheduler/src/cook/components.clj

```diff
@@ -399,6 +416,8 @@
                    role)
      :mesos-framework-name (fnk [[:config [:mesos {framework-name "Cook"}]]]
                              framework-name)
+     :mesos-framework-id (fnk [[:config [:mesos {framework-id nil}]]]
```

Comment:
I think what @shamsimam is saying, and I agree, is that there isn't a reason to define yet another key related to the framework id when we already have `:framework-id`, inside which we can depend on `:config :mesos :framework-id`.